### DISCUSSION TITLE
Feature Request: add ability to show/hide columns in index

### DISCUSSION THREAD

**Discussion by majdghithan** [laravel/nova-issues#5936] [Ideas & Feature Requests]
![Screenshot 2023-10-08 at 20 41 13](https://github.com/laravel/nova-issues/assets/54972678/64cd90eb-5314-4719-9d98-9e8e1295b6d5)

**Comment by milewski** [comment]
It would be nice to have this natively but meanwhile you can do it via this package: https://github.com/dcasia/column-toggler
